google-cloud-datastore/google-cloud-datastore.gemspec

```diff
@@ -19,10 +19,8 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 2.0.0"
 
   gem.add_dependency "google-cloud-core", "~> 0.21.0"
-  gem.add_dependency "grpc", "~> 1.0"
-  gem.add_dependency "google-protobuf", "~> 3.0"
-  gem.add_dependency "googleapis-common-protos", "~> 1.3"
-  gem.add_dependency "google-gax", "~> 0.6.0"
+  gem.add_dependency "google-gax", "~> 0.7.1"
+  gem.add_dependency "googleapis-common-protos", "~> 1.3.5"
```

Comment:
The existing dependency on `googleapis-common-protos` defined by `google-gax` is `~> 1.3.5`. So versions 1.4.0 or 1.5.0 will not resolve.

By specifying `~> 1.3.5` we are just slightly narrowing the acceptable version to ensure that it includes the file permission fix. If google-gax did this for us we would drop the dependency.